src/main/java/io/pinecone/clients/Inference.java

```diff
@@ -0,0 +1,94 @@
+package io.pinecone.clients;
+
+import org.openapitools.control.client.ApiClient;
+import org.openapitools.control.client.ApiException;
+import org.openapitools.control.client.api.InferenceApi;
+import org.openapitools.control.client.model.EmbedRequest;
+import org.openapitools.control.client.model.EmbedRequestInputsInner;
+import org.openapitools.control.client.model.EmbedRequestParameters;
+import org.openapitools.control.client.model.EmbeddingsList;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+/**
+ * The Inference class provides methods to interact with Pinecone's embedding API through the Java SDK. It allows users
```

Comment:
I'd change this wording: "The Inference class provides methods to interact with Pinecone's embedding API" since won't be entirely correct once you add things like `rerank` -- the Inference class actually exposes the _Inference_ API, not the embedding API (it exposes, the embedding _endpoint_ of the Inference API). 

I'd say "The Inference class provides methods to interact with Pinecone's Inference API"

Reply:
Fixed!